src/search/landmarks/landmark_factory_hps_orders.cc

```diff
@@ -0,0 +1,398 @@
+#include "landmark_factory_hps_orders.h"
+
+#include "landmark_graph.h"
+
+#include "util.h"
+
+#include "../option_parser.h"
+#include "../plugin.h"
+
+#include "../utils/logging.h"
+#include "../utils/markup.h"
+
+using namespace std;
+namespace landmarks {
+LandmarkFactoryHPSOrders::LandmarkFactoryHPSOrders(const Options &opts)
+    : lm_factory(opts.get<shared_ptr<LandmarkFactory>>("lm_factory")) {
+}
+
```

Comment:
I think this is one more indicator for the need of a discussion. Currently, we're doing two different things which we both call "landmark factories". On the one hand, it's generating landmarks. On the other hand, it's changing a previous (collection of) landmark graph(s). 
In my opinion, this calls for a redesign on another level. Based on this, I'd rather keep the reasonable orders as an option instead of making this its own landmark factory. (Please don't take this personally, Salomé. I appreciate the work you've put into this so far!)